src/Countries/Morocco.php

```diff
@@ -0,0 +1,153 @@
+<?php
+
+namespace Spatie\Holidays\Countries;
+
+use Carbon\CarbonImmutable;
+use Spatie\Holidays\Concerns\Translatable;
+use Spatie\Holidays\Contracts\HasTranslations;
+use Spatie\Holidays\Exceptions\InvalidYear;
+
+class Morocco extends Country implements HasTranslations
+{
+    use Translatable;
+
+    public function countryCode(): string
+    {
+        return 'ma';
+    }
+
+    public function defaultLocale(): string
+    {
+        return 'en';
```

Comment:
shouldn't this be Arabic by default?

Reply:
thanx for the review 

In fact, I assumed that the Package would be used globally and therefore set English as the default language.